modules/related-posts.php

```diff
@@ -5,7 +5,7 @@
  * First Introduced: 2.9
  * Sort Order: 29
  * Recommendation Order: 9
- * Requires Connection: Yes
+ * Requires Connection: No
```

Comment:
Something to think about as we dev this out -- ideally this should still be yes—it does require a connection to work in any real way, even with this (from my initial look) -- but we want a development-friendly experience. Do we need to rework how "require connection" modules are loaded? Or have a new category "mockable" that represents modules that requires a connection, but have a built-in ability to do _something_ offline.